doc/vim-lsp.txt

```diff
@@ -121,6 +122,12 @@ g:lsp_auto_enable                                       *g:lsp_auto_enable*
 	let g:lsp_auto_enable = 1
 	let g:lsp_auto_enable = 0
 
+g:lsp_hover_win                         *g:lsp_hover_win*
+    Type: String
```

Comment:
Can we use array of string instead.

```vim
let g:lsp_hover_ui = ['float', 'preview']
```

This will allow us to have a consistent pattern when we support other ui.